Mlem/App/Utility/Extensions/Content Models/DeletableProviding+Extensions.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  DeletableProviding+Extensions.swift
+//  Mlem
+//
+//  Created by Sjmarf on 22/07/2024.
+//
+
+import MlemMiddleware
+
+extension DeletableProviding {
+    func toggleDeleted(feedback: Set<FeedbackType>) {
+        if feedback.contains(.toast), !deleted {
+            let task = toggleDeleted()
+            Task {
+                let result = await task.result.get()
```

Comment:
Build failure on Xcode 15
```suggestion
                let result = try await task.result.get()
```